diagnostic.md

```diff
@@ -48,11 +52,26 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    export default Ember.Route.extend({
+      model: function(){
+        return [
+          {
+            movie_id: 123,
+          },
+        ];
+      }
+    });
```

Comment:
Close! The Route model can take a `params` parameter. This gives us access to all route params. Thus, we'd get the movie_id of 123 in the model with `params.movie_id`